### DISCUSSION TITLE
Is there a way to block registration of all service workers?

### DISCUSSION THREAD

**Discussion by crocket** [qutebrowser/qutebrowser#7178] [Q&A] [answered]
On firefox and chromium, I can block all service workers in ublock origin.

On qutebrowser, how can I do that?

**Comment by The-Compiler** [comment, marked as answer]
See [Add settings to block various types of requests · Issue #2626 · qutebrowser/qutebrowser](https://github.com/qutebrowser/qutebrowser/issues/2626) - there's a `ResourceTypeServiceWorker` we could block, I suppose that would be around the same uBlock does, though I haven't checked.

**Reply by crocket** [reply]
Firefox about:config has an option to enable or disable service workers.

In uBlock Origin,

```
*$csp=worker-src 'none'
```

blocks service workers through [ublock origin's csp option](https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#csp).
